client/components/ManageCycles/index.jsx

```diff
@@ -16,7 +17,12 @@ class ManageCycles extends Component {
     }
 
     render() {
-        const { cyclesById } = this.props;
+        const { cyclesById, isAdmin } = this.props;
+
+        if (!isAdmin) {
+            return <Redirect to={{ pathname: '/'}} />;
```

Comment:
We should make a 404 page and redirect here, and block all pages based on access control. I was thinking there ought to be better place to do this but the way we define routes... I couldn't think of a clean way.

Reply:
I added 404 redirects